javascript/first-js-oop-project.md

```diff
@@ -42,103 +42,116 @@ This mini project will help you to learn basic concepts about objects in JavaScr
+const event_obj3 = new Event("Jenny Lewis", "On the line tour 2019");
+```
 1. Create an empty Event array.
-   ```javascript
-   const event_array = new Array();
-   ```
+```javascript
+const event_array = new Array();
+```
 1. Now push the objects created into the array. In order to push objects in the array, do the following.
-   ```javascript
-   // pushing single object to an array
-   event_array.push(event_obj1);
-   // pushing multiple objects to an array at once
-   event_array.push(event_obj1, event_obj2, event_obj3);
-   ```
+```javascript
+// pushing single object to an array
+event_array.push(event_obj1);
+// pushing multiple objects to an array at once
+event_array.push(event_obj1, event_obj2, event_obj3);
+```
 1. Keep checking everytime you push the element using ***console.log***.
-   ```javascript
-   // in order to check whether the elements are pushed, use console.log
-   console.log(event_array);
-   ```
-1. Write a function named `matchTickets` that will take in two values for a price range, and return a list of ticket types available. The method should look like this when it is called:
 ```javascript
-event_obj3.matchedTickets(0, 250)
+// in order to check whether the elements are pushed, use console.log
+console.log(event_array);
 ```
-and will return the string: `Eligible tickets: 1. Balcony 2. Mezzanine` for that particular call. If no tickets are available in that range, it will instead return: `No tickets available.`
 
 1. After you have created the array now we need to write the jQuery code to iterate through it in the same js file event.js .
-  
-   Below is the jQuery code to iterate through the event_array that you have already created.
-    - `.each()` is used to iterate through the array of objects. Above code represents the syntax to iterate using `.each()`
-    - `.html()` is used to return the HTML code from jQuery to the target element of the HTML page.
-    - **event** is the target element in the below code.
-      ```javascript
-      $(document).ready(function() {
-        let html = "";
-        $.each(event_array, function(index, item) {
-          html+= `<li>${item.name} - ${item.description} - ${item.matchedTickets(0, 100)}</li>`;
-        });
-        // insert final html into #event...
-        $("#event").html(html);
-      });
-      ```
-   + Note that all jQuery methods in our examples are inside a document ready event. This is to prevent any jQuery code from running before the document is finished loading (is ready).
+
+Below is the jQuery code to iterate through the event_array that you have already created.
+- `.each()` is used to iterate through the array of objects. Above code represents the syntax to iterate using `.each()`
+- `.html()` is used to return the HTML code from jQuery to the target element of the HTML page.
+- **event** is the target element in the below code.
+```javascript
+$(document).ready(function() {
```

Comment:
how about using arrow functions? (as soon as you use `let` and template literals) 

and identations